lib/phoenix/router.ex

```diff
@@ -72,7 +72,7 @@ defmodule Phoenix.Router do
 
   The URL generated in the named URL helpers is based on the configuration for
   `:url`, `:http` and `:https`. However, if for some reason you need to manually
-  control the URL generation, the url helpers also allow you to pass in a [`URI`](http://elixir-lang.org/docs/stable/elixir/URI.html)
+  control the URL generation, the url helpers also allow you to pass in a [`URI`](https://hexdocs.pm/elixir/URI.html)
```

Comment:
Here we can just use `URI`.

ExDoc will take care of generating the proper URL when docs are built. :)